lib/Lifters/FunctionLifter.cpp

```diff
@@ -440,6 +433,14 @@ llvm::Function *FunctionLifter::LiftFunction(const FunctionDecl &decl) {
     return native_func;
   }
 
+  // Function has no valid instructions.
+  auto &cfg = decl.cfg;
+  if (cfg.find(decl.entry_uid) == cfg.end()) {
+    LOG(WARNING) << "Function missing entry block " << std::hex << decl.address;
+    return native_func;
```

Comment:
i think we should still return null (ie. fail) but only if it's not a decl

if the client wants to declare incoherent functions that should be on the client